### PR TITLE
[SRVCOM-2167] Add Eventing dashboards to Developer View

### PR DESCRIPTION
**Pull request by pierDipi**
As per title.

Notes:
- I had to rename `eventNamespace` variable to just `namespace` so that ODC picks it up from the project (context variable)

Dashboard list:
image

One rendered dashboard:
image

### CODE REVIEW

File: knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml [hunk 1]
```diff
@@ -118,7 +119,7 @@ data:
           "tableColumn": "",
           "targets": [
             {
-              "expr": "sum(rate(mt_broker_ingress_event_count{namespace_name=~\"$eventNamespace\"}[1m]))",
+              "expr": "sum(rate(mt_broker_ingress_event_count{namespace_name=~\"$namespace\"}[1m]))",
```
Comment (skonto) [inline knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml:122]: Interesting could be that this is a convention, I think I hit that before but it is so long ago.

Comment (pierDipi) [reply knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-broker.yaml:122]: Yes, when I applied them initially without this change, it was showing another select option as `eventNamespace` and when renamed to `namespace` it remove that option as it is taking it from the top level project select option

**Comment by skonto** [comment]
In the past odc folks reviewed the dashboards for the developer view. Should we ask them if it is ok to add them?

**Comment by pierDipi** [comment]
We met with the ODC team yesterday and there is no concern to proceed with this change.

/cc @skonto @matzew

**Comment by pierDipi** [comment]
@matzew @skonto can we merge this?

**Review by creydr — COMMENTED** [review]
/lgtm

**Pull request merged**